core-bundle/src/Resources/contao/dca/tl_files.php

```diff
@@ -210,29 +210,29 @@
 		(
 			'label'                   => &$GLOBALS['TL_LANG']['tl_files']['importantPartX'],
 			'inputType'               => 'text',
-			'eval'                    => array('rgxp'=>'natural', 'nospace'=>true, 'tl_class'=>'w50 clr'),
-			'sql'                     => "int(10) NOT NULL default 0"
+			'eval'                    => array('rgxp'=>'digit', 'nospace'=>true, 'tl_class'=>'w50 clr'),
+			'sql'                     => "DOUBLE NOT NULL default 0"
```

Comment:
Can probably be `unsigned`? And we could (start to) use the array syntax?

Reply:
275574db7807e143b3e067d4fe3bea93ee605ee7